pom.xml

```diff
@@ -393,6 +401,14 @@
       </dependencies>
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
Nitpick, it would be nice if you define this value in `<properties>` on top of the file